tests/cli/test_alias.py

```diff
@@ -475,7 +475,7 @@ def test_external_alias_three_args_optional_multiple_regular_help(
         prog_name = Path(sys.argv[0]).name
         expected = inspect.cleandoc(
             f"""\
-            Usage: {prog_name} user-cmd [OPTIONS] [ARG1] ARG2... ARG3
+            Usage: {prog_name} user-cmd [OPTIONS] ARG1 ARG2... [ARG3]
```

Comment:
Args parsing was working correctly only when optional arg was in first position, otherwise `Missing close bracket` was raised